forge-gui/res/cardsfolder/upcoming/ursine_monstrosity.txt

```diff
@@ -0,0 +1,13 @@
+Name:Ursine Monstrosity
+ManaCost:2 G
+Types:Creature Bear Mutant
+PT:3/3
+K:Trample
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | Execute$ TrigMill | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of combat on your turn, mill a card and choose an opponent at random. CARDNAME attacks that player this combat if able. Until end of turn, CARDNAME gains indestructible and gets +1/+1 for each card type among cards in your graveyard.
+SVar:TrigMill:DB$ Mill | Defined$ You | NumCards$ 1 | SubAbility$ DBChoose
+SVar:DBChoose:DB$ ChoosePlayer | Defined$ You | Choices$ Player.Opponent | Random$ True | SubAbility$ DBAnimate
+SVar:DBAnimate:DB$ Animate | Defined$ Self | staticAbilities$ AttackChosen | Duration$ UntilEndOfCombat | SubAbility$ DBPump

```

Comment:
use effect, the ability isn't gained

Reply:
Oh, then Ruhan of the Fomori is wrong...

Reply:
Knight Rampager